Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -155,14 +155,14 @@ private void MutateFloat(ref float val, float min, float max, int bits, int tota
 
         // Starting number of bits that are high, between 0 and bits.
         // In other words, it's val mapped linearly from range [min, max] to range [0, bits], and then rounded.
-        int valInt = (int)MathF.Round((val - min) / (max - min) * bits);
+        int valInt = (int) MathF.Round((val - min) / (max - min) * bits);
```

Comment:
Please revert these changes to all the casts, because we follow the [C# convention on casting](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions).

If you can, please investigate why your IDE complains about this.

Reply:
My .editorconfig file is up to date to the [master branch](https://github.com/space-wizards/space-station-14/blob/master/.editorconfig). I think the relevant setting is `csharp_space_after_cast = true` and it should be adjusted if this is unintended. Also most of the code seems to follow that convention, so are you sure it should be changed?

Reply:
Yes, this should be adjusted. It is likely this way because of the editor config, and people just follow what their IDE tells them without considering the style guide.

Reply:
If you do choose to submit a change to the editorconfig, please make that a separate PR.

Reply:
Spaces removed. I will make a PR for the editorconfig.